packages/strapi-admin/admin/src/components/EventInput/index.js

```diff
@@ -66,7 +64,11 @@ const EventInput = ({ onChange, name: inputName, value: inputValue }) => {
           <tr>
             <td />
             {headersName.map(header => {
-              return <td key={header}>{header}</td>;
+              return (
+                <td key={header}>
+                  <FormattedMessage id={header} defaultMessage={header} />
```

Comment:
Are you sure the `defaultMessage` is correct? (It will display `Settings.webhooks.events.create`) so maybe we can remove it completely or add another defaultMessage